code/modules/mod/modules/modules_maint.dm

```diff
@@ -0,0 +1,247 @@
+//Maint modules for MODsuits
+
+///Springlock Mechanism - allows your modsuit to activate faster, but reagents are very dangerous.
+/obj/item/mod/module/springlock
+	name = "MOD springlock module"
+	desc = "A module that spans the entire size of the MOD unit, sitting under the outer shell. \
+		This mechanical exoskeleton pushes out of the way when the user enters and it helps in booting \
+		up, but was taken out of modern suits because of the springlock's tendency to \"snap\" back \
+		into place when exposed to humidity. You know what it's like to have an entire exoskeleton enter you?"
+	icon_state = "springlock"
+	complexity = 3 // it is inside every part of your suit, so
+	incompatible_modules = list(/obj/item/mod/module/springlock)
+
+/obj/item/mod/module/springlock/on_install()
+	mod.activation_step_time *= 0.75
+
+/obj/item/mod/module/springlock/on_uninstall()
+	mod.activation_step_time /= 0.75
```

Comment:
For something that kills you (instant 500 brute damage) by being exposed to any reagent by touch, vapor or patch application, this feels really underwhelming of a trade-off.

Reply:
the killing is the main focus

Reply:
Would it be possible to make it quick-deploy all the pieces maybe? So instead of toggling each one piece-by-piece, its main advantage is doing all of them AND the small step_time buff?
I do get that killing is the main focus, but aside from it theres really no other function to the mod, it feels weird